Transition.cs

```diff
@@ -0,0 +1,34 @@
+using Godot;
+using System;
+using System.Threading.Tasks;
+
+public class Transition : Control
+{
+    private AnimationPlayer player;
+    private TextureRect texture;
+
+    [Signal] public delegate void TransitionMidPoint();
+
+    [Signal] public delegate void TransitionEnd();
+
+    // Called when the node enters the scene tree for the first time.
+    public override void _Ready()
+    {
+        texture = GetNode<TextureRect>("TransitionTexture");
+        player = GetNode<AnimationPlayer>("TransitionPlayer");
+    }
+
+    public async void Start(Image image)
+    {
+        image.FlipY();
```

Comment:
this comes from "taking a screenshot" of the viewport. it comes flipped on the Y axis, I'm not sure why